examples/common/text_data.py

```diff
@@ -84,81 +88,63 @@ def __init__(self,
                          batch_size=batch_size)
         self.tokenizer_name = tokenizer_name
         self.max_seq_len = max_seq_len
-        self.group_method = group_method
 
         # Build tokenizer
         os.environ['TRANSFORMERS_NO_ADVISORY_WARNINGS'] = '1'
+        os.environ['TOKENIZERS_PARALLELISM'] = 'false'
         self.tokenizer = transformers.AutoTokenizer.from_pretrained(
             self.tokenizer_name)
-        if self.tokenizer._pad_token is None:
-            # Some tokenizers (e.g. GPT2 tokenizer) have no padding token which causes bugs
-            self.tokenizer._pad_token = self.tokenizer._eos_token
-        # suppress warnings when using group_method='concat' and no truncation
+
+        # suppress warnings when using longer 'max_seq_len'
         self.tokenizer.model_max_length = int(1e30)
```

Comment:
is there a reason to set this to `1e30`? instead of using `max_seq_len`

Reply:
I don't think so. It was in here because of on the fly concat.

Reply:
@abhi-mosaic said removing it caused an error, so I propose leaving it for now, rather than investigating.